src/utils/price.ts

```diff
@@ -0,0 +1,77 @@
+import BN from 'bn.js'
+import { assert } from './miscellaneous'
+import { TEN, UNLIMITED_ORDER_AMOUNT_BIGNUMBER } from 'const'
+import BigNumber from 'bignumber.js'
+
+export function adjustAmountToLowerPrecision(amount: BN, higherPrecision: number, lowerPrecision: number): BN {
```

Comment:
Should the params be an object instead of simple params? We had issues in the past with the params not being clear and having not obvious mistakes by entering the params in the wrong order

Reply:
Sure, I'll do that. Hang on for the PR...

Reply:
https://github.com/gnosis/dex-react/pull/438